R/data.R

```diff
@@ -27,26 +27,33 @@
 ##' @param trim_pillar2 The number of days of pillar 2 data to trim to avoid
 ##'   back-fill issues. We typically use a value of 7 days.
 ##'
+##' @param adm_backfill_date A date string, representing the date that we start
+##'   using age-aggregated admissions from the PHE dashboard for England NHS
+##'   regions instead of admissions by age from the SUS linelist (which are used
+##'   before this date). This is to account for backfill, and should be
+##'   carefully updated with every new SUS data delivery.
```

Comment:
Sorry, looking below adm_backfill_date is the last date we use admissions by age rather than the first date we use age-aggregated admissions.

So maybe better to say something like "A date string, representing the last date we use admissions by age from the SUS linelist for England NHS regions. After this date we use age-aggregated admissions from the UKHSA dashboard. This is to......"